bindings/rust/standard/integration/src/network/tls_client.rs

```diff
@@ -46,38 +46,17 @@ mod kms_pq {
     // supports ML-KEM.
     #[test_log::test(tokio::test)]
     async fn pq_handshake() -> Result<(), Box<dyn std::error::Error>> {
-        let policy = Policy::from_version("KMS-PQ-TLS-1-0-2020-07")?;
+        let policy = Policy::from_version("PQ-TLS-1-2-2023-10-09")?;
```

Comment:
Just to confirm: you can't use "default_pq" here because "default_pq" has ML-KEM instead of Kyber?

Reply:
Yes, `default_pq` contains ML-KEM (and only ML-KEM, not Kyber). `PQ-TLS-1-2-2023-10-09` contains both ML-KEM and Kyber. KMS currently only has Kyber deployed, so we can't use `default_pq` yet.